integration-tests/helpers/use-db.js

```diff
@@ -62,7 +62,7 @@ const DbTestUtil = {
       await manager.query(`DELETE
                            FROM "${entity.tableName}";`)
     }
-    if (connectionType === "sqlite") {
+    if (dataSourceType === "sqlite") {
       await manager.query(`PRAGMA foreign_keys = ON`)
     } else {
       await manager.query(`SET session_replication_role = 'origin';`)
```

Comment:
self-review: @riqwan I think we need to call `destroy` on db as well here. wdyt?

Reply:
teardown seems to be different from shutdown. Lets keep them separate. 

Teardown is done on `afterEach`, whereas shutdown is done `afterAll`